apps/pyth-lazer-agent/src/jrpc_handle.rs

```diff
@@ -240,6 +249,33 @@ async fn handle_push_update<T: AsyncRead + AsyncWrite + Unpin>(
     }
 }
 
+async fn handle_batch_push_update<T: AsyncRead + AsyncWrite + Unpin>(
+    sender: &mut Sender<T>,
+    lazer_publisher: &LazerPublisher,
+    batch_request: &[FeedUpdateParams],
+    request_id: JrpcId,
+) -> anyhow::Result<()> {
+    for request_params in batch_request.iter() {
+        if let Err(err) = lazer_publisher
+            .push_feed_update(request_params.clone().into())
+            .await
+        {
+            debug!("error while sending updates: {:?}", err);
+            send_json(
+                sender,
+                &JrpcErrorResponse {
+                    jsonrpc: JsonRpcVersion::V2,
+                    error: JrpcError::SendUpdateError(request_params.clone()).into(),
+                    id: request_id.clone(),
+                },
+            )
+            .await?;
+            anyhow::bail!("Error processing batch update: {:?}", err);
```

Comment:
I think this `bail!` may not be needed here. It's not present in `handle_push_update`. Looks like it will result in client receiving two error responses.